src/software/amazon/ion/system/IonWriterBuilder.java

```diff
@@ -97,4 +99,15 @@
      * @return the IVM minimizing strategy.
      */
     public abstract IvmMinimizing getIvmMinimizing();
+
+    /**
+     * Builds a new writer based on this builder's configuration
+     * properties.
+     *
+     * @param out the stream that will receive Ion data.
+     * Must not be null.
+     *
+     * @return a new {@link IonWriter} instance; not {@code null}.
+     */
+    public abstract IonWriter build(OutputStream out);
```

Comment:
Since no existing code could previously invoke IonWriterBuilder.build(), all of the code should be invocations of Ion(Text|Binary)WriterBuilder.build(), which is still a concrete method implemented in both subclasses, so there's no binary compatibility issue.